recipes/rbdl-casadi/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rbdl-casadi" %}
+{% set version = "2.6.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/rbdl-casadi/archive/{{ version }}.tar.gz
+  sha256: e07ab94e25ca99fdfa77341f6ea5bda5480bac3493e7af03158144da71c6a6eb
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+  rpaths:
```

Comment:
What's the reason for these?

Reply:
Casadi is currently unavaible on conda-forge. But this is I my plan to produce the recipe for it. So for now, I must skip windows because of a missing dependency.

For OSX, this is because I had trouble compiling on osx, but I've just seen workaround yesterday, I'll test them today, hopefully I can compile on OSX.

Reply:
I meant the list of `rpaths` with this commit. I made a separate one for the the skip.

Reply:
I double checked and rbdl installs directly in lib, so adjusting rpaths is probably useless